com.unity.testing.hdrp/TestRunner/HDRP_TestSettings.cs

```diff
@@ -49,7 +49,7 @@ void Awake()
 
             // Render pipeline is only reconstructed when a frame is renderer
             // If scene requires lightmap baking, we have to force it
-            if (forceCameraRenderDuringSetup && !Application.isPlaying)
+            if (forceCameraRenderDuringSetup && !Application.isPlaying && Application.platform != RuntimePlatform.OSXEditor)
```

Comment:
I would put a comment on here so we remember to remove this when the original issue is fixed